dmn-compatibility/src/app.js

```diff
@@ -17,18 +18,18 @@ async function importXML(xml) {
   xml = await migrateDiagram(xml);
 
   // (1.2) import DMN 1.3 diagram
-  dmnModeler.importXML(xml, err => {
-    if (err) {
-      console.error(err);
-    }
-  });
+  try {

```

Comment:
I pretty much like that we finally get rid of the mixed behaviors from the import + the migration API 👍